### PR TITLE
44 review accessibility

### LINKED ISSUE #44: Review accessibility
Research how to improve our web accessibility. Search for a validation page.

1. Images needs to be served in next gen format.
2. Remove the header video on low resolutions for better performance.
3.  Change colors from the navigation bar to make more contrast.

### PR DESCRIPTION
**Pull request by juanjoseortega-aircury**
Testing with lightouse doesn't work, it get bugged when removing the video

### CODE REVIEW

File: assets/custom.css [hunk 1]
```diff
@@ -2,6 +2,9 @@
     --color: #0f477e;
     --circle: 150px;
 }
+.greedy-nav a, .text a {
+    color: #11579C;
```
Comment (cristobaldiaz-aircury) [inline assets/custom.css]: Hard coded color

**Review by cristobaldiaz-aircury — CHANGES_REQUESTED** [review]
Pending checking rest of accessibility issues, using lighthouse or not.

File: _includes/page__hero.html [hunk 2]
```diff
@@ -2,11 +2,6 @@
 
 <div class="page__hero--overlay home_hero" style="background: {{ page.header.overlay_color | default: 'white' }}">
     <div class="background_media" style="{% if page.header.image %}background-image: url('{{page.header.image | relative_url}}'){% endif %}">
-        {% if page.header.video %}
```
Comment (cristobaldiaz-aircury) [inline _includes/page__hero.html]: Ideally this video should be removed only for small resolutions

**Review by cristobaldiaz-aircury — CHANGES_REQUESTED** [review]
1. Low contrast issues at main page:
image
2. There's room to improve performance, a ticket can be created for that.

File: _includes/page__hero.html (hunk 2 above)
Comment (juanjoseortega-aircury) [reply _includes/page__hero.html]: This can be removed in the ticket #46 to improve the performance at mobiles

**Review by cristobaldiaz-aircury — APPROVED** [review]

**Pull request merged**